src/index.html

```diff
@@ -22,6 +22,293 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      ></a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <a
+            href="#"
+            class="header__nav-list-item is-active"
+          >
+            Apple
+          </a>
+          <a
+            href="#"
+            class="header__nav-list-item"
+          >
+            Samsung
+          </a>
+          <a
+            href="#"
+            class="header__nav-list-item"
+          >
+            Smartphones
+          </a>
+          <a
+            href="#"
+            class="header__nav-list-item"
+            data-qa="nav-hover"
+          >
+            Laptops & Computers
+          </a>
+          <a
+            href="#"
+            class="header__nav-list-item"
+          >
+            Gadgets
+          </a>
+          <a
+            href="#"
+            class="header__nav-list-item"
+          >
+            Tablets
+          </a>
+          <a
+            href="#"
+            class="header__nav-list-item"
+          >
+            Photo
+          </a>
+          <a
+            href="#"
+            class="header__nav-list-item"
+          >
+            Video
+          </a>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__picture"></div>
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <h4 class="card__code">Product code: 195434</h4>
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="review">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <h4 class="card__price-title">Price:</h4>
+          <h4 class="card__price-tag">$2,199</h4>
+        </div>
+        <button
+          class="card__button"
+          type="submit"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```

Reply:
Still not fixed